src/payment/strategies/credit-card-redirect/credit-card-redirect-payment-strategy.ts

```diff
@@ -0,0 +1,65 @@
+import { FormPoster } from '@bigcommerce/form-poster';
+import { some } from 'lodash';
+
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { RequestError } from '../../../common/error/errors';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import { OrderFinalizationNotRequiredError } from '../../../order/errors';
+import { PaymentArgumentInvalidError } from '../../errors';
+import PaymentActionCreator from '../../payment-action-creator';
+import { PaymentRequestOptions } from '../../payment-request-options';
+import * as paymentStatusTypes from '../../payment-status-types';
+import PaymentStrategy from '../payment-strategy';
+
+export default class CreditCardRedirectPaymentStrategy implements PaymentStrategy {
```

Comment:
can you extend instead `CreditCardPaymentStrategy` ? Eventually we will be moving towards iframed credit card fields, and that functionality comes for free when you extend `CreditCardPaymentStrategy`

Reply:
I can see this was marked as resolved but don't see the new code, maybe you forgot to push your latest changes?

Reply:
Maybe something went wrong with github, I've squashed the commits  and pushed again